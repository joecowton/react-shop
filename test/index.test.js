import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';

import headerBlock from '../src/index.js';

// test ('Render headerblock to page', () => {
//   const headerDiv = renderer.create(
//     <div class="headerDiv"></div>
//   );
//
//   let header = headerDiv.toJSON();
//   expect(header).toMatchSnapshot();
//
//
// });

describe ('Render Header block', () => {
  const wrapper = shallow(<headerBlock />);
  // expect(wrapper.find(headerBlock).render().find('.headerDiv')).to.have.length(1);
  expect(wrapper.contains(<div className="headerDiv"></div>)).to.equal(true);
})
