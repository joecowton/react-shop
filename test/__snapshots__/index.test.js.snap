// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render headerblock to page 1`] = `
<div
  class="headerDiv"
/>
`;
